postman/Authorization.postman_collection.json

```diff
@@ -0,0 +1,227 @@
+					"cookie": [],
+					"body": "{\r\n    \"email\": \"example@mail.ru\",\r\n    \"password\": \"strongPassword1_\",\r\n    \"firstName\": \"Ivan\",\r\n    \"lastName\": \"Ivanov\"\r\n}"
+				}
+			]
+		},
+		{
+			"name": "Sign in",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "07b173b6-98ce-4fbf-bbaa-9348d3d53844",
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n    \"email\": \"example@mail.ru\",\r\n    \"password\": \"strongPassword1_\"\r\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{auth_url}}/signIn",
+					"host": [
+						"{{auth_url}}"
+					],
+					"path": [
+						"signIn"
+					]
+				}
+			},
+			"response": [
+				{
+					"name": "SignIn Example",
+					"originalRequest": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"email\": \"\",\r\n    \"password\": \"\"\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{auth_url}}/signIn",
+							"host": [
+								"{{auth_url}}"
+							],
+							"path": [
+								"signIn"
+							]
+						}
+					},
+					"_postman_previewlanguage": null,
+					"header": null,
+					"cookie": [],
+					"body": "{\r\n    \"email\": \"example@mail.ru\",\r\n    \"password\": \"strongPassword1_\"\r\n}"
+				}
+			]
+		},
+		{
+			"name": "Refresh token",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "a7c745e0-d8ee-416f-9c91-aceb84de53ca",
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "{{auth_token}}",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{auth_url}}/refresh?refreshToken={{refresh_token}}",
+					"host": [
+						"{{auth_url}}"
+					],
+					"path": [
+						"refresh"
+					],
+					"query": [
+						{
+							"key": "refreshToken",
+							"value": "{{refresh_token}}"
+						}
+					]
+				}
+			},
+			"response": []
+		}
+	],
+	"event": [
+		{
+			"listen": "prerequest",
+			"script": {
+				"id": "24ff9b2f-1295-48ce-a024-4e2e71b7186f",
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		},
+		{
+			"listen": "test",
+			"script": {
+				"id": "7bb17ab2-daf9-4adb-a166-1a1e5281eb9c",
+				"type": "text/javascript",
+				"exec": [
+					"if (pm.response.code === 200) {",
+					"    var headers = pm.response.headers;",
+					"    pm.environment.set(\"auth_token\", headers.get(\"Access-Token\"));",
+					"    pm.environment.set(\"refresh_token\", headers.get(\"Refresh-Token\"));",
+					"}"
+				]
+			}
+		}
+	],
+	"protocolProfileBehavior": {}
+}
```

Comment:
Turn on in idea settings ```Ensure an empty line at the end of a file on Save``` to get rid of these red warnings

Reply:
Thanks for noticing that, Fixed